src/lib/features/playground/playground-service.ts

```diff
@@ -68,37 +68,31 @@ export class PlaygroundService {
                     ? new Date(context.currentTime)
                     : undefined,
             };
-            const output: PlaygroundFeatureSchema[] = await Promise.all(
-                client
-                    .getFeatureToggleDefinitions()
-                    .map(async (feature: FeatureInterface) => {
-                        const strategyEvaluationResult: FeatureStrategiesEvaluationResult =
-                            client.isEnabled(feature.name, clientContext);
+            const output: PlaygroundFeatureSchema[] = client
+                .getFeatureToggleDefinitions()
+                .map((feature: FeatureInterface) => {
+                    const strategyEvaluationResult: FeatureStrategiesEvaluationResult =
+                        client.isEnabled(feature.name, clientContext);
 
-                        const isEnabled =
-                            strategyEvaluationResult.result === true &&
-                            feature.enabled;
+                    const isEnabled =
+                        strategyEvaluationResult.result === true &&
+                        feature.enabled;
 
-                        return {
-                            isEnabled,
-                            isEnabledInCurrentEnvironment: feature.enabled,
-                            strategies: {
-                                result: strategyEvaluationResult.result,
-                                data: strategyEvaluationResult.strategies,
-                            },
-                            projectId:
-                                await this.featureToggleService.getProjectId(
-                                    feature.name,
-                                ),
-                            variant: client.getVariant(
-                                feature.name,
-                                clientContext,
-                            ),
-                            name: feature.name,
-                            variants: variantsMap[feature.name] || [],
-                        };
-                    }),
-            );
+                    return {
+                        isEnabled,
+                        isEnabledInCurrentEnvironment: feature.enabled,
+                        strategies: {
+                            result: strategyEvaluationResult.result,
+                            data: strategyEvaluationResult.strategies,
+                        },
+                        projectId:
+                            features.find((f) => f.name === feature.name)
```

Comment:
features list is fetched upfront and has all project ids

Reply:
Yeah, this seems sensible. According to the type in `src/lib/types/stores/feature-strategies-store.ts`, "project" is always a string, though. Do you think it'll ever be empty? But then again, I guess this is JS, so you never really know 💁🏼  

Just one suggestion: if this is to help with performance, would it make sense to also convert the features list into an object/map and access it with `features[feature.name]` instead of linearly searching through the list again and again? Especially if the list of features is long, we'll end up with a O(n) instead of O(nm) (where n = number of features and m = number of client features), right?